### DISCUSSION TITLE
Is there a link on the dashboards to report bugs and give feedback, like this?

### DISCUSSION THREAD

**Discussion by mattcorey** [TelemetryDeck/Viewer#448] [Ideas]
I had an idea I wanted to post while looking at my dashboards, but I couldn't find any link to report a bug or give feedback.  I knew there was one in the footer of a webpage somewhere -- I ended up having to type 'telemetrydeck.com' to get to the main page, and found the link there.  Would be great to have that a standard footer on the dashboard pages as well!  (and probably should have the copyright, terms and privacy links as well?)

**Comment by winsmith** [comment]
Fantastic idea, thanks!
